samples/Exporters/AspNet/Global.asax.cs

```diff
@@ -16,7 +17,7 @@ protected void Application_Start()
         {
             this.openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
                 (builder) => builder.AddDependencyInstrumentation()
-                .AddRequestInstrumentation()
+                .AddRequestInstrumentation((opt) => opt.TextFormat = new TraceContextFormatActivityCijo())
```

Comment:
Did you mean to check in this file? I don't see `TraceContextFormatActivityCijo` defined anywhere.

Reply:
no! thanks for catching it